translate/storage/properties.py

```diff
@@ -478,11 +479,8 @@ def encoding(self):
             return self.personality.default_encoding
 
     def __str__(self):
-        """Convert to a string. Double check that unicode is handled
-        somehow here."""
-        source = self.getoutput()
-        assert isinstance(source, six.text_type)
-        return source.encode(self.encoding)
```

Comment:
A minor correction, `@encoding` is actually just above this `__str__()`.

Yes, there are more `encoding` and `encode` on the unit.